docker-compose.yml

```diff
@@ -1,258 +1,261 @@
-      - MULESOFT_API_CURRENCY_EXCHANGE_RATE_SECRET
-      - MULESOFT_API_EXPOSURE_PERIOD_URL
-      - MULESOFT_API_UKEF_MDM_EA_URL
-      - MULESOFT_API_UKEF_MDM_EA_KEY
-      - MULESOFT_API_UKEF_MDM_EA_SECRET
-      - MULESOFT_API_UKEF_ESTORE_EA_URL
-      - MULESOFT_API_UKEF_ESTORE_EA_KEY
-      - MULESOFT_API_UKEF_ESTORE_EA_SECRET
-      - AZURE_ACBS_FUNCTION_URL=http://host.docker.internal:7071
-      - GOV_NOTIFY_API_KEY
-    entrypoint: npx nodemon src/index.js
-
-  gef-ui:
-    build: ./gef-ui
-    image: dtfs/gef-ui
-    restart: always
-    depends_on:
-      - redis
-      - portal-api
-    ports:
-      - "5006:5006"
-    volumes:
-      - ./gef-ui/scripts:/app/scripts:ro
-      - ./gef-ui/styles:/app/styles:ro
-      - ./gef-ui/templates:/app/templates:ro
-      - ./gef-ui/server:/app/server:ro
-    environment:
-      - DEAL_API_URL=http://portal-api:5001
-      - COMPANIES_HOUSE_API_URL=https://api.companieshouse.gov.uk
-      - COMPANIES_HOUSE_API_KEY
-      - REFERENCE_DATA_PROXY_URL=http://reference-data-proxy:5002
-      - SESSION_SECRET
-      - REDIS_HOSTNAME=redis
-    command: node dist/server.js
-  
-  # ACBS function is causing an out of memory error on github actions. disable for now
-  # acbs-function:
-  #   build: ./azure-functions/acbs-function
-  #   image: dtfs/azure-function-acbs
-  #   restart: always
-  #   ports:
-  #     - "7071:80"
-  #   environment:
-  #     - AzureWebJobsStorage=DefaultEndpointsProtocol=https;AccountName=${AZURE_PORTAL_STORAGE_ACCOUNT};AccountKey=${AZURE_PORTAL_STORAGE_ACCESS_KEY};EndpointSuffix=core.windows.net
-  #     - WEBSITE_HOSTNAME=localhost:7071
-  #     - MULESOFT_API_KEY
-  #     - MULESOFT_API_SECRET
-  #     - MULESOFT_API_UKEF_TF_EA_URL
-  #     - MULESOFT_API_UKEF_MDM_EA_URL
-  #     - MULESOFT_API_UKEF_MDM_EA_KEY
-  #     - MULESOFT_API_UKEF_MDM_EA_SECRET
-
-  redis:
-    image: "redis:alpine"
-    ports:
-      - "6379:6379"
-
-  reverse-proxy:
-    build: ./reverse-proxy
-    image: "dtfs/reverse-proxy"
-    depends_on:
-      - portal
-      - gef-ui
-    ports:
-      - "80:80"
-    environment:
-      - PORTAL_UI=portal:5000
-      - GEF_UI=gef-ui:5006
-
-  test-hook-api:
-    build: ./test-hook-api
-    image: dtfs/test-hook-api
-    restart: always
-    depends_on:
-      - portal-api
-    ports:
-      - "5069:5069"
-    volumes:
-      - ./test-hook-api/src:/app/src:ro
-    environment:
-      - PORT=5069
-      - CORS_ORIGIN=http://localhost:5000
-      - AZURE_WORKFLOW_STORAGE_ACCOUNT
-      - AZURE_WORKFLOW_STORAGE_ACCESS_KEY
-      - AZURE_WORKFLOW_FILESHARE_NAME
-      - AZURE_WORKFLOW_EXPORT_FOLDER
-      - AZURE_WORKFLOW_IMPORT_FOLDER
-      - AZURE_PORTAL_STORAGE_ACCOUNT
-      - AZURE_PORTAL_STORAGE_ACCESS_KEY
-      - AZURE_PORTAL_FILESHARE_NAME
-      - AZURE_PORTAL_EXPORT_FOLDER
-      - DTFS_PORTAL_SCHEDULER
-      - GOV_NOTIFY_API_KEY
-      - GOV_NOTIFY_EMAIL_RECIPIENT
-
-    entrypoint: npx nodemon src/index.js
+version: "3"

```

Comment:
Just wondering why the whole file has been marked as a diff

Reply:
This was the wonderful world of windows line endings. Despite setting up my environment properly it converted these to CRLF. Resolved now. Will watch for it